src/main/java/com/adyen/service/management/TerminalSettingsTerminalLevel.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Management API
+ *
+ * The version of the OpenAPI document: 1
+ * Contact: developer-experience@adyen.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+package com.adyen.service.management;
+
+import com.adyen.ApiKeyAuthenticatedService;
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.constants.ApiConstants;
+import com.adyen.model.management.*;
```

Comment:
Looks like there are some superfluous imports here. can we lose this line in the service classes?

Reply:
Fixed in 7c3dd95c0bf4b09196721004203338682a73847c.